### PR TITLE
api-docs: add examples if present

### PR DESCRIPTION
**Pull request by inian**
Depends on https://github.com/supabase/supabase/pull/8523 (at least lesser merge conflicts in generated files)

Before
CleanShot 2022-08-19

After
CleanShot 2022-08-19

### CODE REVIEW

File: about/src/css/custom.css [hunk 1]
```diff
@@ -1,5 +1,5 @@
-/* 
-  Font import 
+/*
+  Font import
 */
 
 @import url('https://rsms.me/inter/inter.css');
```
Comment (kiwicopple) [inline about/src/css/custom.css:5]: Where did this one come from? I think we were using Circular previously

File: about/src/css/custom.css [hunk 2]
```diff
@@ -1036,6 +1036,7 @@ h4.method-list-item-label .method-list-item-label-name {
   padding: var(--ifm-code-padding-vertical) 0;
 }
 h4.method-list-item-label .method-list-item-label-badge {
+  overflow-wrap: anywhere;
```
Comment (kiwicopple) [inline about/src/css/custom.css:1039]: what does this one do?

File: apps/reference/_api/usage.mdx [hunk 3]
```diff
@@ -430,6 +430,7 @@ GET https://api.supabase.com/v1/projects/{ref}/functions
         
         </span>
```
Comment (kiwicopple) [inline apps/reference/_api/usage.mdx:431]: we should move this file to a `generated` folder at some point (will make it easier to .gitignore it when we're ready)

**Review by kiwicopple — APPROVED** [review]

File: about/src/css/custom.css (hunk 2 above)
Comment (inian) [reply about/src/css/custom.css:1039]: The jwt example overflows into the sidebar since it is a very long token, unless i add this in

**Pull request merged**